documentation/src/docs/quickstart.md

```diff
@@ -0,0 +1,51 @@
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+
+# Getting Started

```

Comment:
We have both a quickstart and a getting started document, should we consolidate them? I don't see a reason for having both, but granted that the content is different.

Reply:
IMO, we can leave them separate since `quickstart` is meant for providing guidance on how to build Flink applications using the connector whereas getting started document provides instructions on how to setup/build connector from IDE.